python/pickle/PyROOT_readtests.py

```diff
@@ -165,6 +165,21 @@ def test2TBufferCheck( self ):
       f1 = TBufferFile( TBuffer.kWrite )
       f2 = pickle.loads( pickle.dumps( f1 ) )
 
+   def test_return_facade(self):
```

Comment:
Can you follow the naming scheme of the other two tests (e.g. test3ReturnFacade)?

Reply:
Right I missed that sorry! I'll wait to see the results from the CI then I'll change the name :+1: